sdc/src/sdc/sdc.d

```diff
@@ -75,14 +75,29 @@ final class SDC {
 		semantic.terminate();
 		backend.visit(semantic.buildMain(modules));
 	}
-	
-	void codeGen(string objFile) {
+
+	void runLLVMPasses() {
```

Comment:
I can rename `runLLVMPasses` to `codeGen`. I think it is clearest if main.d tells the backend to compile, codegen, and output stuff (instead of have codegen output a file).
Tell me how you want it, I have no preference.